ddtrace/internal/agent.py

```diff
@@ -50,6 +78,6 @@ def get_connection(url, timeout=DEFAULT_TIMEOUT):
     elif parsed.scheme == "unix":
         conn = UDSHTTPConnection(parsed.path, parsed.scheme == "https", hostname, parsed.port, timeout=timeout)
     else:
-        raise ValueError("Unknown agent protocol %s" % parsed.scheme)
+        raise ValueError("Unsupported protocol '%s'" % parsed.scheme)
```

Comment:
You can always do `"%r".format(thing)` to get it quoted... assuming it is a string type of course.

non-blocking comment, just random feedback.